File: IT/Strutture dati/Esercizi/C++/esercizio7.cpp
/*
file esercizio7.cpp
compilare con: g++ -std=c++11 esercizio7.cpp
*/

#include <iostream>
#include <iterator>

using std::cout;
using std::endl;
using std::ostream_iterator;
using std::begin;
using std::end;

int main()
{
	int nums[] = {1, 3, 5, 7, 9};

	cout << "[";
	copy(begin(nums), end(nums), ostream_iterator<int>(cout, ", "));
	cout << "\b\b]" << endl;

	int tot = 0;

	for (int n : nums) {
		tot += n;
	}

	cout << "Somma degli elementi: " << tot << endl;
}

File: IT/1/Strutture dati/Esercizi/Java/Esercizio7.java
/* file Esercizio7.java */

import java.util.Arrays;

public class Esercizio7 {
	public static void main(String [] args) {
		int [] nums = {1, 3, 5, 7, 9};

		System.out.println(Arrays.toString(nums));

		int tot = 0;

		for (int n : nums) {
			tot += n;
		}

		System.out.println("Somma degli elementi: " + tot);
	}
}

